docs/administration/management/managing-users/index.md

```diff
@@ -34,22 +36,48 @@ Token for user aa37d63d-3186-45d5-aa40-5d221cb170bf (new.web.user@example.org) g
 Token: c0fa47fa00ae4013a13fd7n
 ```
 
-## Revoke access tokens
+### Revoke access tokens
 
 ```console
 $ kubectl exec -i -t deployment/reana-server -- flask reana-admin token-revoke --email new.web.user@example.org --admin-access-token $REANA_ACCESS_TOKEN
 User token c0fa47fa00ae4013a13fd7n (new.web.user@example.org) was successfully revoked.
 ```
 
-## Export users
+### Export users
 
 ```console
 $ kubectl exec -i -t deployment/reana-server -- flask reana-admin user-export --admin-access-token $REANA_ACCESS_TOKEN > myusers.csv
 ```
 
-## Import users
+### Import users
 
 ```console
 $ # put myusers.csv onto the node in the /var/reana directory and run:
 $ kubectl exec -i -t deployment/reana-server -- flask reana-admin user-import --admin-access-token $REANA_ACCESS_TOKEN --file /var/reana/myusers.csv
 ```
+
+## User registration via sign up form
+
+By exposing the REANA User Interface, local users sing up and sign in
+are enabled by default. When accessing the UI for the first time, you
+will be prompted with a sign in form and a link to the sign up form.
+
+![ui-sign-in](../../../images/ui-sign-in.png)
+
+As an administrator, you can configure REANA to disable the local users
+or to hide the sign up form. Please check the [Configuring user access](../../configuration/configuring-access) page to see how.
+
+## User registration via Single Sign-On
+
+Handling of users with Single Sign-On (SSO) is also possible for CERN
+deployments. This configuration can be combined with local users or used
+exclusively. When accessing the UI you will see a page like this:
+
+![ui-sso](../../../images/ui-sso.png)
+
+When clicking on "Sign in with SSO" you will be redirected to the
```

Comment:
Nice catch that we had a brief "Configuring user access section" already. Two thoughts upon seeing the changes:

- Looking at screenshots and the wording such as "when clicking on SSO..." etc, this is less for managing users than for explaining the whole user access in case SSO is enabled. Considering that, I think that the screenshots and the explanatory text may be better located in the "Configuring user access part" after all? We would present there the three options (default + disable sign-up + enable SSO) with screenshots and meaning what each is, plus how to enable/disable/configure by the admins.  And then here, on this page, we could basically just say a quick note for each command e.g. for `create` we could just say "If you have enabled SSO access, then user creation via CLI is not necessary since users will be created via SSO when they first log in".  This might read better in my eyes. WDYT?

- "Managing users" page stays a bit alone on the lhs menu amongst "Deployment" and "Configuration" menu sections.  Shall we add a new menu section "Management" and attach "Managing users" page under Management there, for consistency? Then we could add other sections such as "Managing disk space" there more easily in the future.

Reply:
- Makes sense, I've changed it. Let me know what you think.
- Agree, index Management page created.